dist/caveat.d.ts

```diff
@@ -0,0 +1,71 @@
+import { CaveatOptions, Satisfier } from './types';
```

Comment:
I found that I couldn't use the library by including the github link in another project's deps unless I included the dist directory. Since merging this in and publishing it to npm was premature, I had to add the dist file. I can remove it as soon as this is approved.

Reply:
Good catch. Looks like the npm scripts need to be a little more... forgiving? robust? I think a `prepare` script should avoid this problem in the future [reference](https://blog.jim-nielsen.com/2018/installing-and-building-an-npm-package-from-github/). I'll add that in a separate PR that removes these as well